src/main/java/frc/robot/subsystems/CoralShooter/CoralShooter.java

```diff
@@ -21,10 +21,6 @@ public void setDutyCycle(double speed) {
     io.setDutyCycle(speed);
   }
 
-  public boolean getOuttakeSensor() {
```

Comment:
We can keep this, just make it refer to the `outtakeSensor` in the inputs on the IO hardware class